src/Bicep.Core.Samples/Files/Completions/declarations.json

```diff
@@ -167,6 +167,204 @@
       "newText": "param ${1:Identifier} string {\n  secure: true\n}"
     }
   },
+  {
```

Comment:
I'm going to assume you're considering this a temporary state of affairs, and that these will later be built up automatically from individual .bicep files, so I won't complain about the format.